{"uid":"53c00cede0bf7a1e","name":"잘못된 ID,PW 입력 시 로그인이 실패하는지 확인","fullName":"src.tests.ui_tests.test_ui_login#test_login","historyId":"98e8f149c7227721782a965f730ba74a","time":{"start":1746450802392,"stop":1746450810416,"duration":8024},"status":"failed","statusMessage":"Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\n  (Session info: chrome=136.0.7103.59); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55f33e8c67fa <unknown>\n#1 0x55f33e36ae90 <unknown>\n#2 0x55f33e3bc870 <unknown>\n#3 0x55f33e3bca61 <unknown>\n#4 0x55f33e40acb4 <unknown>\n#5 0x55f33e3e226d <unknown>\n#6 0x55f33e408135 <unknown>\n#7 0x55f33e3e2013 <unknown>\n#8 0x55f33e3aeb3b <unknown>\n#9 0x55f33e3af7a1 <unknown>\n#10 0x55f33e88bb9b <unknown>\n#11 0x55f33e88fa8a <unknown>\n#12 0x55f33e873912 <unknown>\n#13 0x55f33e890604 <unknown>\n#14 0x55f33e85874f <unknown>\n#15 0x55f33e8b4678 <unknown>\n#16 0x55f33e8b4856 <unknown>\n#17 0x55f33e8c5666 <unknown>\n#18 0x7f7e382721f5 <unknown>","statusTrace":"src/tests/ui_tests/test_ui_login.py:23: in test_login\n    login_password_input = driver.find_element(By.ID, \"pw\")\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:917: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:448: in execute\n    self.error_handler.check_response(response)\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=136.0.7103.59); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55f33e8c67fa <unknown>\nE   #1 0x55f33e36ae90 <unknown>\nE   #2 0x55f33e3bc870 <unknown>\nE   #3 0x55f33e3bca61 <unknown>\nE   #4 0x55f33e40acb4 <unknown>\nE   #5 0x55f33e3e226d <unknown>\nE   #6 0x55f33e408135 <unknown>\nE   #7 0x55f33e3e2013 <unknown>\nE   #8 0x55f33e3aeb3b <unknown>\nE   #9 0x55f33e3af7a1 <unknown>\nE   #10 0x55f33e88bb9b <unknown>\nE   #11 0x55f33e88fa8a <unknown>\nE   #12 0x55f33e873912 <unknown>\nE   #13 0x55f33e890604 <unknown>\nE   #14 0x55f33e85874f <unknown>\nE   #15 0x55f33e8b4678 <unknown>\nE   #16 0x55f33e8b4856 <unknown>\nE   #17 0x55f33e8c5666 <unknown>\nE   #18 0x7f7e382721f5 <unknown>\n\nDuring handling of the above exception, another exception occurred:\nsrc/tests/ui_tests/test_ui_login.py:37: in test_login\n    pytest.fail(f\"❌ 로그인 실패 :{str(e)}\")\nE   Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=136.0.7103.59); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55f33e8c67fa <unknown>\nE   #1 0x55f33e36ae90 <unknown>\nE   #2 0x55f33e3bc870 <unknown>\nE   #3 0x55f33e3bca61 <unknown>\nE   #4 0x55f33e40acb4 <unknown>\nE   #5 0x55f33e3e226d <unknown>\nE   #6 0x55f33e408135 <unknown>\nE   #7 0x55f33e3e2013 <unknown>\nE   #8 0x55f33e3aeb3b <unknown>\nE   #9 0x55f33e3af7a1 <unknown>\nE   #10 0x55f33e88bb9b <unknown>\nE   #11 0x55f33e88fa8a <unknown>\nE   #12 0x55f33e873912 <unknown>\nE   #13 0x55f33e890604 <unknown>\nE   #14 0x55f33e85874f <unknown>\nE   #15 0x55f33e8b4678 <unknown>\nE   #16 0x55f33e8b4856 <unknown>\nE   #17 0x55f33e8c5666 <unknown>\nE   #18 0x7f7e382721f5 <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\n  (Session info: chrome=136.0.7103.59); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55f33e8c67fa <unknown>\n#1 0x55f33e36ae90 <unknown>\n#2 0x55f33e3bc870 <unknown>\n#3 0x55f33e3bca61 <unknown>\n#4 0x55f33e40acb4 <unknown>\n#5 0x55f33e3e226d <unknown>\n#6 0x55f33e408135 <unknown>\n#7 0x55f33e3e2013 <unknown>\n#8 0x55f33e3aeb3b <unknown>\n#9 0x55f33e3af7a1 <unknown>\n#10 0x55f33e88bb9b <unknown>\n#11 0x55f33e88fa8a <unknown>\n#12 0x55f33e873912 <unknown>\n#13 0x55f33e890604 <unknown>\n#14 0x55f33e85874f <unknown>\n#15 0x55f33e8b4678 <unknown>\n#16 0x55f33e8b4856 <unknown>\n#17 0x55f33e8c5666 <unknown>\n#18 0x7f7e382721f5 <unknown>","statusTrace":"src/tests/ui_tests/test_ui_login.py:23: in test_login\n    login_password_input = driver.find_element(By.ID, \"pw\")\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:917: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:448: in execute\n    self.error_handler.check_response(response)\n/usr/local/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=136.0.7103.59); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55f33e8c67fa <unknown>\nE   #1 0x55f33e36ae90 <unknown>\nE   #2 0x55f33e3bc870 <unknown>\nE   #3 0x55f33e3bca61 <unknown>\nE   #4 0x55f33e40acb4 <unknown>\nE   #5 0x55f33e3e226d <unknown>\nE   #6 0x55f33e408135 <unknown>\nE   #7 0x55f33e3e2013 <unknown>\nE   #8 0x55f33e3aeb3b <unknown>\nE   #9 0x55f33e3af7a1 <unknown>\nE   #10 0x55f33e88bb9b <unknown>\nE   #11 0x55f33e88fa8a <unknown>\nE   #12 0x55f33e873912 <unknown>\nE   #13 0x55f33e890604 <unknown>\nE   #14 0x55f33e85874f <unknown>\nE   #15 0x55f33e8b4678 <unknown>\nE   #16 0x55f33e8b4856 <unknown>\nE   #17 0x55f33e8c5666 <unknown>\nE   #18 0x7f7e382721f5 <unknown>\n\nDuring handling of the above exception, another exception occurred:\nsrc/tests/ui_tests/test_ui_login.py:37: in test_login\n    pytest.fail(f\"❌ 로그인 실패 :{str(e)}\")\nE   Failed: ❌ 로그인 실패 :Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"pw\"]\"}\nE     (Session info: chrome=136.0.7103.59); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE   Stacktrace:\nE   #0 0x55f33e8c67fa <unknown>\nE   #1 0x55f33e36ae90 <unknown>\nE   #2 0x55f33e3bc870 <unknown>\nE   #3 0x55f33e3bca61 <unknown>\nE   #4 0x55f33e40acb4 <unknown>\nE   #5 0x55f33e3e226d <unknown>\nE   #6 0x55f33e408135 <unknown>\nE   #7 0x55f33e3e2013 <unknown>\nE   #8 0x55f33e3aeb3b <unknown>\nE   #9 0x55f33e3af7a1 <unknown>\nE   #10 0x55f33e88bb9b <unknown>\nE   #11 0x55f33e88fa8a <unknown>\nE   #12 0x55f33e873912 <unknown>\nE   #13 0x55f33e890604 <unknown>\nE   #14 0x55f33e85874f <unknown>\nE   #15 0x55f33e8b4678 <unknown>\nE   #16 0x55f33e8b4856 <unknown>\nE   #17 0x55f33e8c5666 <unknown>\nE   #18 0x7f7e382721f5 <unknown>","steps":[],"attachments":[{"uid":"49340e949bcf6861","name":"Failure Screenshot","source":"49340e949bcf6861.png","type":"image/png","size":162159},{"name":"stdout","type":"text/plain","size":0}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"예외 케이스"},{"name":"story","value":"잘못된 ID,PW 입력"},{"name":"parentSuite","value":"src.tests.ui_tests"},{"name":"suite","value":"test_ui_login"},{"name":"host","value":"0c9ccf202096"},{"name":"thread","value":"15-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"src.tests.ui_tests.test_ui_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"53c00cede0bf7a1e.json","parameterValues":[]}